brain_games/games/prime.py

```diff
@@ -0,0 +1,19 @@
+from typing import Dict
+
+from brain_games.games.utils import generate_game_round_data, get_random_number
+from sympy import isprime
+
+RULES = 'Answer "yes" if given number is prime. Otherwise answer "no".'
+
+
+def prime() -> Dict[str, str]:
+    """
+    Forms brain-prime game round data
+
+    :return: brain-prime game round data
+    :rtype: Dict[str, str]
+    """
+    number = get_random_number(1, 100)
+    correct_answer = "yes" if isprime(number) else "no"
```

Comment:
do you really need this function to check if number from 1 to 100 is prime? can it be done in a simpler way?